sound/effect.h

```diff
@@ -362,6 +362,7 @@ EFFECT(pwroff);
 EFFECT(clash);
 EFFECT(force);  // also polyphonic
 EFFECT(stab);   // also polyphonic
+EFFECT(spin);
```

Comment:
Why add spin to implement slash?

Reply:
Was experimenting with using X acceleration to trigger stabs but pulled it out for this merge.  Missed the spin declaration in effects.h.  Will remove.

Reply:
removed.